nsv13/code/controllers/subsystem/overmap_mode.dm

```diff
@@ -361,6 +367,51 @@ SUBSYSTEM_DEF(overmap_mode)
 	if(mode.difficulty <= 0)
 		mode.difficulty = 1
 
+/datum/controller/subsystem/overmap_mode/proc/force_mode(new_mode)
+	if(!mode || !mode_initialised)
+		mode = new_mode
+		forced_mode = new_mode
+		return initial(mode.name)
+	if(mode_initialised)
+		QDEL_NULL(mode)
+		mode = new new_mode()
+		forced_mode = mode
+		mode_initialised = FALSE //Reset this in case something fails after it.
+		setup_overmap_mode() //we will have to reinitialize manually
+		return mode.name
+
+/datum/controller/subsystem/overmap_mode/proc/toggle_hardmode()
+	if(!initialized) //Not yet
+		message_admins("Hardmode failed to activate! Turn it on when the overmap gamemode subsystem is done loading in the control panel!")
+		return FALSE
+	hard_mode_enabled = !hard_mode_enabled
+	log_game("Hard mode has been [hard_mode_enabled ? "enabled" : "disabled"]!")
+	if(hard_mode_enabled) //Go on, clear the map out
+		SSresearch.hardmode_tech_enable()
+		mode.objectives += new /datum/overmap_objective/clear_system/rubicon
+		mode.objectives += new /datum/overmap_objective/clear_system/dolos
```

Comment:
This causes roundstart overmap mode setup to runtime because the objectives are already initialized before the roundstart initialization, as opposed to being a typepath like the others.

Reply:
Yep. fixing that